twint-sdk/build.gradle

```diff
@@ -0,0 +1,143 @@
+configurations.maybeCreate("default")
+artifacts.add("default", file('TwintSdk-android-7.0.0.aar'))
```

Comment:
This looks a bit scary, but this is the only way I could make publishing work.

At first I included the AAR in the twint module, but this causes the AAR to be nested inside of the AAR created when publishing the twint module. Nested (aka fat) AARs are not supported. Instead, we now publish the twint AAR as a separate dependency

Reply:
Let's discuss this more before merging